overlord/snapstate/snapstate_update_test.go

```diff
@@ -8442,38 +8392,18 @@ func (s *snapmgrTestSuite) TestUpdateBaseKernelSingleRebootUnsupportedWithCoreHa
 		}
```

Comment:
this is for UC16, on master tasks for different snaps are in separate lanes, here they end up in one additional shared lane, I think we want to be conservative and avoid that for UC16

Reply:
Good point, I'm now not doing any of the lane changes for UC16